### PR TITLE
docs: Add Node.js install steps to the readme

### PR DESCRIPTION
**Pull request by francocorreasosa**
## Description

I've added a pre-install step to the Gatsby CLI in the readme. Telling people they need to get node.js installed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,14 +95,20 @@ Gatsby is a modern web framework for blazing fast websites.
 
 You can get a new Gatsby site up and running on your local dev environment in 5 minutes with these four steps:
 
-1. **Install the Gatsby CLI.**
+1. Install Node.js
+
+In order to execute the Gatsby CLI (command-line interface) you will need to have [Node.js](https://nodejs.org/) installed on your machine. Node.js comes with `npm` and `npx` pre-installed.
```
Comment (marcysutton) [inline README.md]: I think a better approach would be to add a note about Node being a prerequisite, and centralize the install instructions on a single doc on gatsbyjs.org so we can keep it updated. Here's an issue related to that: https://github.com/gatsbyjs/gatsby/issues/15529

File: docs/docs/quick-start.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ This quick start is intended for intermediate to advanced developers. For a gent
 
 **Note**: this video uses `npx`, which is a tool to execute an npm package without first installing it. Running the command `npx gatsby new` is the same as running `gatsby new` after installing the gatsby-cli on your computer.
 
+### Install Node.js
```
Comment (marcysutton) [inline docs/docs/quick-start.md]: Same as above. I'd recommend adding a note about prerequisites and centralizing the install instructions in the Preparing Your Environment section: https://www.gatsbyjs.org/docs/preparing-your-environment/

**Comment by LekoArts** [comment]
Hi @francocorreasosa 👋 Would you be up for implementing the requested changes from Marcy? Namely moving the instructions to gatsbyjs.org/docs/preparing-your-environment and referencing this document in the docs you currently have changed?

**Comment by francocorreasosa** [comment]
@LekoArts  Yes, will do that later today. Thanks

**Comment by LekoArts** [comment]
Hi! We're closing this PR since it went stale. If you want to continue working on this please let us know. Thanks!

**Pull request closed without merging**